src/routes/quickstarts/react-native/react-native-testing-library.mdx

```diff
@@ -0,0 +1,242 @@
+# Mock Network Requests in React Native Testing Library with Mirage
+
+Use your Mirage server to test your React Native application under different server scenarios using [React Native Testing Library](https://github.com/testing-library/native-testing-library) and [Jest Native](https://github.com/testing-library/jest-native).
+
+> This is a quickstart guide should also work for people using [`react-test-renderer`](https://reactjs.org/docs/test-renderer.html) or [`react-native-testing-library`](https://github.com/callstack/react-native-testing-library).
+
+## Step 1: Install Mirage
+
+First, make sure you have Mirage installed:
+
+```bash
+# Using npm
+npm install --save-dev miragejs
+
+# Using Yarn
+yarn add --dev miragejs
+```
+
+## Step 2: Setup your testing environment
+
+Out of the box, React Native does a good job to polyfill browser API's that we might need in libraries that were originally built to run on the browser. Our problem when using Mirage in a Jest testing environment is that we don't have access to some properties that the browser provides by default. To get around this issue we must update Jest's environment immediately after its setup is finished.
+
+To update your Jest environment follow these steps:
+
+1. Create a setup file in the root of your project. For the purpose of this guide we'll call it `jest.setup.js`.
```

Comment:
I think you can remove this step and start right off with the installing of xmlhttprequest. Then in the Patch variable step you can say...

"Create a new `jest.setup.js` file in the root of your project. We'll use this file to patch the variable global...."

Reply:
👍